ogr_fdw_common.h

```diff
@@ -15,6 +15,7 @@
 #include <ctype.h>
 #include "stringbuffer.h"
 
+#define MAX_IDENTIFIER_LEN 63+1
```

Comment:
Is there a reason for this mass re-naming? And I think there's a false economy in shortening the buffer... I just liked having it around as a generic buffer. Is the maximum identifier length a "thing", is it a PgSQL limitation? If so there's probably a #define lying around you can pull from a postgres header instead.

Reply:
Yes, it's a postgresql limit. I've now added the appropriate include file and referenced the define NAMEDATALEN (see https://www.postgresql.org/docs/12/sql-syntax-lexical.html)